.github/workflows/consume-release.yml

```diff
@@ -6,13 +6,44 @@ on:
   workflow_run:
     workflows: [Release]
     types: [completed]
+    branches: [develop]
+  workflow_dispatch:
+    inputs:
+      workflowRunId:
+        description: The workflow_run.id of the release workflow that has published artifacts
+        required: true
 
 jobs:
-  on-success:
+  get-images:
```

Comment:
We should only get artifacts if we were successful in publishing them, it might be easier to merge this job into the next one and use the same job execution conditional